test/spec/unit/reject.js

```diff
@@ -0,0 +1,61 @@
+import { remote } from '../../../index'
+import { tmpdir } from 'os'
+
+describe('rejection', () => {
+    before(async function() {
+        this.client = remote({
+            screenshotPath: tmpdir()
+        })
+
+        const createSession = global.mock('post', '/session')
+        await this.client.init()
+        createSession.isDone().should.be.true
+    })
+
+    it('should not take screenshot if rejection handler is provided', function () {
```

Comment:
I know.
This is because of Babel issue: https://github.com/babel/babel/pull/3608. (We are parsing stack traces, but they are broken now). You can ping that guys if you want faster resolution.

@christian-bromann  we can disable these test, while Babel feature is open. It affects only `babel-register`, so users will not notice anything